### DISCUSSION TITLE
How to interact with my document editor and invoke the editor's global search function.

### DISCUSSION THREAD

**Discussion by THL1234** [ONLYOFFICE/sdkjs-plugins#179] [Q&A] [answered]
I want to interact with my document editor and invoke the editor's global search function. How can I use the callCommon or executeMethod method to achieve this requirement？
the code doesn't work

    this.connector.callCommand(function() {
      var oDocument = Api.GetDocument();
      var aSearch = oDocument.Search("xxx"); // xxx is someting to search
      aSearch[0].SetBold(true);
    }, function() { console.log("callback command"); });

![搜素](https://user-images.githubusercontent.com/80459759/226267730-ee1035d2-cb2d-460d-89ea-5fd7ed97db1d.png)

**Comment by trofim24** [comment, marked as answer]
We've added [Search](https://api.onlyoffice.com/docs/office-api/usage-api/text-document-api/ApiDocument/Methods/Search/) and [SearchAndReplace](https://api.onlyoffice.com/docs/office-api/usage-api/text-document-api/ApiDocument/Methods/SearchAndReplace/) methods. Thanks
